src/components/common/SocialMediaContainer/social-media-container.scss

```
@@ -0,0 +1,26 @@
+.social-media-container {
+  background: $blue;
+  position: absolute;
+  right: 0rem;
```

When positioning elements you should always set (`top` or `bottom`) AND (`right` or `left`). Check it in other positioned elements below too.
Also, when values are `0`, no unit is needed.
```suggestion
  right: 0;
  top: 0;
```